content/learn/book/migration-guides/0.9-0.10/_index.md

```diff
@@ -0,0 +1,837 @@
+</div>
+
+Rename `ScalingMode::Auto` to `ScalingMode::AutoMin` if you are using it.
+
+### [Change `From<Icosphere>` to `TryFrom<Icosphere>`](https://github.com/bevyengine/bevy/pull/6484)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+</div>
+
+```rust
+// 0.9
+shape::Icosphere {
+    radius: 0.5,
+    subdivisions: 5,
+}
+.into()
+
+// 0.10
+shape::Icosphere {
+    radius: 0.5,
+    subdivisions: 5,
+}
+.try_into()
+.unwrap()
+```
+
+### [Directly extract joints into SkinnedMeshJoints](https://github.com/bevyengine/bevy/pull/6833)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+    <div class="migration-guide-area-tag">Animation</div>
+</div>
+
+`ExtractedJoints` has been removed. Read the bound bones from `SkinnedMeshJoints` instead.
+
+### [Intepret glTF colors as linear instead of sRGB](https://github.com/bevyengine/bevy/pull/6828)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+    <div class="migration-guide-area-tag">Assets</div>
+</div>
+
+No api changes are required, but it's possible that your gltf meshes look different
+
+### [The `update_frame_count` system should be placed in CorePlugin](https://github.com/bevyengine/bevy/pull/6676)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+    <div class="migration-guide-area-tag">Core</div>
+    <div class="migration-guide-area-tag">Time</div>
+</div>
+
+The `FrameCount`  resource was previously only updated when using the `bevy_render` feature. If you are not using this feature but still want the `FrameCount` it will now be updated correctly.
+
+### [Migrate engine to Schedule v3](https://github.com/bevyengine/bevy/pull/7267)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+- Calls to `.label(MyLabel)` should be replaced with `.in_set(MySet)`
+- Stages have been removed. Replace these with system sets, and then add command flushes using the `apply_system_buffers` exclusive system where needed.
+- The `CoreStage`, `StartupStage,`RenderStage`and`AssetStage`enums have been replaced with`CoreSet`,`StartupSet, `RenderSet` and `AssetSet`. The same scheduling guarantees have been preserved.
+  - Systems are no longer added to `CoreSet::Update` by default. Add systems manually if this behavior is needed, although you should consider adding your game logic systems to `CoreSchedule::FixedTimestep` instead for more reliable framerate-independent behavior.
+  - Similarly, startup systems are no longer part of `StartupSet::Startup` by default. In most cases, this won’t matter to you.
+  - For example, `add_system_to_stage(CoreStage::PostUpdate, my_system)` should be replaced with
+  - `add_system(my_system.in_set(CoreSet::PostUpdate)`

```

Comment:
I think this needs to be `.in_base_set` now

Reply:
@IceSentry this comment might have been overlooked. For `CoreSet`, you need to use `in_base_set`

Reply:
```suggestion
  - `add_system(my_system.in_base_set(CoreSet::PostUpdate)`

```